lib/huginn_agent/spec_runner.rb

```diff
@@ -34,6 +34,10 @@ def bundle
         shell_out "cp .env spec/huginn"
       end
       Dir.chdir('spec/huginn') do
+        if !File.exists?('.env')
+          shell_out "cp .env.example .env"
+          shell_out "SECRET=$(dd if=/dev/urandom bs=36 count=1 | openssl base64); sed -i .orig \"s:REPLACE_ME_NOW\\!:$SECRET:\" .env"
```

Comment:
I am not sure if it is safe to assume every system has `openssl` and `sed` installed. We could do the replacement in ruby or skip it since I don't know if the secret token is relevant for running the specs.

Reply:
Yea, I agree. Removing it.